itests/markets_v1_deal_test.go

```diff
@@ -58,6 +62,70 @@ func TestMarketsV1Deal(t *testing.T) {
 	assertFilesEqual(t, inPath, outPath)
 }
 
+func TestMarketsV1OfflineDeal(t *testing.T) {
+	// Create a CAR file
+	//tempdir := t.TempDir()
+	tempdir := tempHome
+	log.Debugw("using tempdir", "dir", tempdir)
+
+	rseed := 0
+	size := 7 << 20 // 7MiB file
+
+	inPath, err := testutil.CreateRandomFile(tempdir, rseed, size)
+	require.NoError(t, err)
+	res, err := f.fullNode.ClientImport(f.ctx, lapi.FileRef{Path: inPath})
+	require.NoError(t, err)
+
+	// Get the piece size and commP
+	rootCid := res.Root
+	pieceInfo, err := f.fullNode.ClientDealPieceCID(f.ctx, rootCid)
+	require.NoError(t, err)
+
+	// Create a new markets v1 deal
+	dp := f.DefaultMarketsV1DealParams()
+	dp.Data.Root = res.Root
+	// Replace with params for manual storage deal (offline deal)
+	dp.Data.TransferType = storagemarket.TTManual
+	dp.Data.PieceCid = &pieceInfo.PieceCID
+	dp.Data.PieceSize = pieceInfo.PieceSize.Unpadded()
+	dp.VerifiedDeal = true
+
+	log.Debugw("starting offline deal", "root", res.Root)
+	dealProposalCid, err := f.fullNode.ClientStartDeal(f.ctx, &dp)
+	require.NoError(t, err)
+	log.Debugw("got deal proposal cid", "cid", dealProposalCid)
+
+	// Wait for the deal to reach StorageDealCheckForAcceptance on the client
+	cd, err := f.fullNode.ClientGetDealInfo(f.ctx, *dealProposalCid)
+	require.NoError(t, err)
+	require.Eventually(t, func() bool {
+		cd, _ := f.fullNode.ClientGetDealInfo(f.ctx, *dealProposalCid)
+		return cd.State == storagemarket.StorageDealCheckForAcceptance
```

Comment:
This test is still blocked, as deal stays in `StorageDealReserveClientFunds`. Not sure just yet, why we are not passing this state/stage.